tf2_sensor_msgs/tf2_sensor_msgs/__init__.py

```diff
@@ -26,4 +26,4 @@
 # ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
 # POSSIBILITY OF SUCH DAMAGE.
 
-from tf2_sensor_msgs import *  # noqa(E401)
+from .tf2_sensor_msgs import *  # noqa(E401)
```

Comment:
Do you want to be importing everything into this scope? I think `transform_points` and `do_transform_cloud` are the only public API right?

Maybe consider something like:
```suggestion
from .tf2_sensor_msgs import do_transform_cloud
from .tf2_sensor_msgs import transform_points

__all__ = [
    'do_transform_cloud',
    'transform_points',
]
```